src/Widgets/VPN/VPNMenuItem.vala

```diff
@@ -16,106 +16,135 @@
  */
 
 public class Network.VPNMenuItem : Gtk.ListBoxRow {
-    public signal void user_action ();
-    public NM.RemoteConnection? connection { get; set; }
+    public signal void connect_clicked ();
```

Comment:
Can we use names that are more associated with the functionality here since this doesn't only get triggered when these buttons are pressed?

Reply:
Oops, I guess I read this wrong! It looks like this ultimately end up calling NetworkManager to `activate_connection`. So maybe `activate` and `deactivate` would be better signal names here.

I think the fact that it's a button click is kind of an internal implementation detail of this class and it's not really the concern of `VPNPage` (or any other class) that a button was involved.

Reply:
Can't use activate here `error: Signals with the same name as a signal in a base type are not supported` .It is because ListBoxRow already has this signal

Reply:
How about `activate_vpn` and `deactivate_vpn`